bin/dbuild

```diff
@@ -0,0 +1,19 @@
+#!/bin/zsh
+
+project_dir=$(pwd)
+mkdir -p ./bin/logs
+
+logfile="$project_dir"/bin/logs/"$1".log
+if [ -f "$logfile" ]; then
+  rm "$logfile"
+fi
+
+git clone git@github.com:hmcts/"$1".git &>> "$logfile"
```

Comment:
How much space is this going to add to a persons and where is this being cloned into?

Reply:
None, the repo is deleted after the image is built and log file is recreated on each build so it will always be quite short